package.json

```diff
@@ -20,6 +20,7 @@
   },
   "homepage": "https://github.com/hiyaryan/the-cdj#readme",
   "devDependencies": {
+    "@types/jsonwebtoken": "^9.0.7",
```

Comment:
Did you mean to install this in the root package.json file or did you mean to install it into backend/package.json?